shelley/chain-and-ledger/formal-spec/transactions.tex

```diff
@@ -133,12 +133,12 @@ \section{Transactions}
       \var{txbody}
       & \TxBody
       & \begin{array}{l}
-        \powerset{\TxIn} \times (\Ix \mapsto \TxOut) \times \seqof{\DCert}
+        \powerset{\TxIn} \times \seqof{\TxOut} \times \seqof{\DCert}
         \times \Coin \times \Slot \times \Wdrl
         \\ ~~~~\times \Update \times \MetaDataHash^?
         \end{array}
       \\
-      \var{wit} & \TxWitness & (\VKey \mapsto \Sig, \HashScr \mapsto \Script)
+      \var{wit} & \TxWitness & (\VKey \mapsto \Sig, \HashScr \mapsto \Script) (NSF: Slightly to the scripts being referred to as ``witnesses''.)
```

Comment:
i am not sure what you mean?

Reply:
Sorry, typo in my comment. I meant to say "I'm slightly surprised to see the scripts also being referred to as _witnesses_." It was just a little jarring/awkward, not really a problem.

Reply:
"validators" is probably a better name for the scripts used in this way, but I'm not sure if that name works for signatures.